torch/csrc/distributed/rpc/rpc_agent.h

```diff
@@ -130,7 +171,7 @@ class TORCH_API RpcAgent {
 
   // Stop accepting requests and shutdown the RPC framework as soon as possible
   // by terminating all RPC threads.
-  virtual void shutdown() = 0;
+  virtual void shutdown();
```

Comment:
This no longer needs to be virtual, as implementations would override `shutdownImpl`?